build.sbt

```diff
@@ -21,8 +20,9 @@ lazy val root = (project in file("."))
   .settings(
     organization := "com.lightbend.play",
     name := "play-socket-io",
-    crossScalaVersions := Seq(scala212, scala211),
-    scalacOptions ++= Seq("-feature", "-Xfatal-warnings", "-target:jvm-1.8"),
+    scalaVersion := scala212,
```

Comment:
Make 2.13 the default. It will also be consistent with some of the other projects below.

Reply:
Done